librz/io/p/io_zip.c

```diff
@@ -370,9 +370,83 @@ char *rz_io_zip_get_by_file_idx(const char *archivename, const char *idx, ut32 p
 	return filename;
 }
 
+static char *find_ipa_binary(const char *filename, int rw, int mode) {
+	RzList *files = NULL;
```

Comment:
Check for filename here - could it be NULL or not?

Reply:
no it can't